code_examples/workshop/attester/generateAccount.ts

```diff
@@ -1,47 +1,34 @@
 import { config as envConfig } from 'dotenv'
 
-import { cryptoWaitReady, mnemonicGenerate } from '@polkadot/util-crypto'
-import { Keyring } from '@polkadot/api'
+import { mnemonicGenerate, mnemonicToMiniSecret } from '@polkadot/util-crypto'
 
 import * as Kilt from '@kiltprotocol/sdk-js'
 
-export function generateAccount(keyring: Keyring): {
+export function generateAccount(mnemonic?: string): {
```

Comment:
Would this be more straightforward?
```suggestion
export function generateAccount(mnemonic = mnemonicGenerate()): {
```